core/src/main/java/org/elasticsearch/common/breaker/CircuitBreaker.java

```diff
@@ -32,6 +32,7 @@
     String FIELDDATA = "fielddata";
     String REQUEST = "request";
     String IN_FLIGHT_REQUESTS = "in_flight_requests";
+    String BUCKET = "bucket";
```

Comment:
Yeah, it also allows someone to tweak the limit and overhead separately if it doesn't estimate the usage correctly